gpkit/tests/t_constraints.py

```diff
@@ -59,6 +60,19 @@ def test_equality_relaxation(self):
         m2 = Model(rc.relaxvars.prod() * x**0.01, rc)
         self.assertAlmostEqual(m2.solve(verbosity=0)(x), 3, 5)
 
+    def test_sp_relaxation(self):
+        x = Variable('x')
+        y = Variable('y')
+        z = Variable('z')
+        with SignomialsEnabled():
+            m = Model(x, [x+y >= z], {z:2})
+        r1 = ConstantsRelaxed(m)
+        r2 = ConstraintsRelaxed(m)
+        r3 = ConstraintsRelaxedEqually(m)
+        self.assertEqual(len(r1.varkeys), 5)
+        self.assertEqual(len(r2.varkeys), 4)
+        self.assertEqual(len(r3.varkeys), 4)
```

Comment:
maybe maybe add another constraint to the model so that r2 and r3 have a different number of varkeys?